modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/MDFRequestForm/steps/Review/components/Body/components/ActivityReviewEntry/ActivityReviewEntry.tsx

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import {TypeActivityExternalReferenceCode} from '../../../../../../../../common/enums/typeActivityExternalReferenceCode';
+import useSelectedTypeActivity from '../../../../../../../../common/hooks/useSelectedTypeActivity';
+import MDFRequestActivity from '../../../../../../../../common/interfaces/mdfRequestActivity';
+import useGetTypeActivities from '../../../../../../../../common/services/liferay/object/type-activities/useGetTypeActivities';
+import useGetActivitiesName from '../../../../Hooks/useGetActivitiesName';
+import useGetTacticsName from '../../../../Hooks/useGetTacticsName';
+import ActivityContent from './components/ActivityContent';
+import ContentMarket from './components/ContentMarket';
+import DigitalMarket from './components/DigitalMarket';
+import Event from './components/Event';
+import MiscellaneousMarket from './components/MiscellaneousMarket';
+
+interface IProps {
+	values: MDFRequestActivity;
+}
+
+type TypeOfActivityComponent = {
+	[key in string]?: JSX.Element;
+};
+
+const ActivityReviewEntry = ({values}: IProps) => {
```

Comment:
```suggestion
const ActivityReviewEntry = ({mdfRequestActivity}: IProps) => {
```

Reply:
The variables and props names need to be clear to read. Check this approach in all files. Let's avoid use just values or value.